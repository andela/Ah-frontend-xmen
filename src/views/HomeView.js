import React from 'react';
import App from '../components/LandingPage/Banner';
import Articles from '../components/LandingPage/Articles';

const HomeView = () => (
  <div>
    <App />
    <Articles />
  </div>
);

export default HomeView;
