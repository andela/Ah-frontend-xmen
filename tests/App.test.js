import React from 'react';
import { shallow } from 'enzyme';
import HomeView from '../src/views/HomeView';

describe('app tests', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<HomeView />);
    expect(wrapper).toMatchSnapshot();
  });
});
