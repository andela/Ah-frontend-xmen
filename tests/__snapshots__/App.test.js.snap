// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app tests should render without crashing 1`] = `ShallowWrapper {}`;
